### ISSUE TITLE
Watch doesnt receive motion notifications

### ISSUE THREAD

**Issue by esecules** [iancmcc/ouimeaux#119]
Ive been trying to get the watch.py example to work with both my wemo switch and motion. The script is able to detect both of them, but when I toggle the switch, or move around I dont see the statechange receiver fire. 

What should I be seeing on wireshark? I do see a HTTP SUBSCRIBE from my desktop to my wemo which are given a 200 OK by the wemo. I also see a few NOTIFY's coming out of the wemo but they dont  correspond with movement or switch activity. 

This is the NOTIFY activity I see coming out of the switch. I dont see any directed at my desktop directly though.
```
NOTIFY * HTTP/1.1
HOST: <multicast-ip>:<multicast-port>
CACHE-CONTROL: max-age=86400
LOCATION: http://<wemo-switch-ip>:49154/setup.xml
OPT: "http://schemas.upnp.org/upnp/1/0/"; ns=01
01-NLS: <some long hash>
NT: urn:Belkin:service:manufacture:1
NTS: ssdp:alive
SERVER: Unspecified, UPnP/1.0, Unspecified
X-User-Agent: redsonic
USN: uuid:Socket-1_0-221241K0101C2C::urn:Belkin:service:manufacture:1
``` 

I have tried running this on my ubuntu desktop, and raspberry pi with the same result. I have a wemo switch and a wemo motion. Which I can control and check the status of without issue.

**Comment by esecules** [comment]
The root cause is that the server that is supposed to listen to UDP traffic is not started. I believe it should be the server from [subscribe.py:95](https://github.com/iancmcc/ouimeaux/blob/b5f33439c51d100da55a07cb33a19662c32a0c3b/ouimeaux/subscribe.py#L95) I am not familiar enough with the project to know where to start this server.

**Comment by esecules** [comment]
ok so I found this commit (https://github.com/iancmcc/ouimeaux/commit/1aacda0d4b0a86ddd9b4cf9bf8e259e149b4459f) which fixed #63, but in the process it made it impossible to open the device discovery server and the notification server at the same time. This probably worked before caching was disabled?

Anyway, @iancmcc, would this change need to be reverted, or do I need two environments, one for discovery and one for notifications and they share devices somehow? Or is it possible to repurpose an environment for notifications after discovery is finished?
